.github/pull_request_template.md

```diff
@@ -1,19 +0,0 @@
-<!--
-Thanks for your contribution! Take a moment to answer these questions so that reviewers have the information they need to properly understand your changes:
-
-* What is the current state of things and why does it need to change?
-* What is the solution your changes offer and how does it work?
-
-Are there any issues or other links reviewers should consult to understand this pull request better? For instance:
-
-* Fixes #12345
-* See: #67890
--->
-
-## Examples
```

Comment:
Typically I leave the template, but delete this section. It's intended to be generally useful above here

Reply:
Makes sense!